### PR TITLE
WebRTC: Create an Access Token

### PR DESCRIPTION
**Pull request by weroh**

### CODE REVIEW

File: lib/PortingPhoneNumber.php [hunk 1]
```diff
@@ -9,7 +9,18 @@
  */
 class PortingPhoneNumber extends ApiResource
 {
-    const OBJECT_NAME = "porting_phone_number";
+    const OBJECT_NAME = "number_order_phone_number";
```
Comment (d-telnyx) [inline lib/PortingPhoneNumber.php:12]: @weroh this was an error on our end and should remain as `porting_phone_number` the tests will fail until we fix the OpenAPI spec and Mock :(

Comment (weroh) [reply lib/PortingPhoneNumber.php]: OK I have reverted this change in the latest update

**Review by d-telnyx — APPROVED** [review]

File: lib/TelephonyCredential.php [hunk 2]
```diff
@@ -27,4 +27,41 @@ public static function classUrl()
...
+        $requestor = new \Telnyx\ApiRequestor($opts->apiKey, $baseUrl);
+
+        // Instead of calling the neatly packaged request() function, we're calling _requestRaw()
+        list($rbody, $rcode, $rheaders, $myApiKey) = $requestor->_requestRaw('post', $url, $params, $headers);
+
+        // Remove headers that don't need to be persistent across other requests
+        $opts->discardNonPersistentHeaders();
+
+        if ($rcode < 200 || $rcode >= 300) { // If there is an error then we need to interpret as a json and return the error object
+            $json = $requestor->_interpretResponse($rbody, $rcode, $rheaders);
```
Comment (d-telnyx) [inline lib/TelephonyCredential.php:58]: Should `_interpretResponse` be made public in [APIRequestor](https://github.com/team-telnyx/telnyx-php/pull/69/files#diff-4946db3aaa038c0ffbcbd3f8635f772c9c7163a2d407afbe3022742deca82d4eL380)

Comment (weroh) [reply lib/TelephonyCredential.php:58]: _interpretResponse() is now public

**Pull request merged**